roles/ccp/tasks/main.yaml

```diff
@@ -0,0 +1,9 @@
+- include: 110_copy_artifacts.yaml
+
+- include: 120_install_ccp.yaml
+
+- include: 130_render_ccp_configuration.yaml
+
+- include: 140_build_oss_images.yaml
```

Comment:
Is that possible to make this optional and be able to skip this step through setting some variable like `build_oss_images=False (True by default) in group_vars?